tests/utilities/test_auto_restart.py

```diff
@@ -0,0 +1,588 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import math
+import os
+import random
+from collections.abc import Iterable
+from typing import Optional
+
+import numpy as np
+import pytest
+import torch
+import torch.distributed as dist
+import torch.multiprocessing as mp
+from torch.utils.data import BatchSampler, DistributedSampler, RandomSampler, SequentialSampler
+from torch.utils.data._utils.worker import get_worker_info
+from torch.utils.data.dataloader import DataLoader, default_collate
+from torch.utils.data.dataset import Dataset, IterableDataset
+
+import tests.helpers.utils as tutils
+from pytorch_lightning import seed_everything
+from pytorch_lightning.utilities.auto_restart import CaptureIterativeDataset, FastForwardSampler
```

Comment:
```suggestion
from pytorch_lightning.utilities.auto_restart import CaptureIterativeDataset, FastForwardSampler
```
Class names have changed, tests would fail

Reply:
Done !